src/plugin.js

```diff
@@ -2,6 +2,9 @@ function initPlugin() {
   const onFileInput = function(editor, file) {
     // We use TinyMCE's editor.dom here (rather than document.createElement) as it is empirically more reliable within the editor
     const img = editor.dom.create('img', {id: `tinymce-new-image-${file.name}`});
+    if (!img && honeybadger) {
```

Comment:
Lowercase honeybadger?

Reply:
Ah. . .good catch. Should be a capital h.